plugins/shared/info-panel/index.js

```diff
@@ -295,8 +295,12 @@ class InfoPanel {
                     e.preventDefault();
                     e.stopPropagation();
 
-                    // TODO: This attempts to scroll to fixed elements
-                    $(document).scrollTop(error.$el.offset().top - 80);
+                    // Add tabIndex if element is not currently focusable
+                    if(!error.$el.is(axs.utils.FOCUSABLE_ELEMENTS_SELECTOR)){
```

Comment:
What if it already has a tabindex?

also nit: space around parens `if (...) {`